docs/configuration_file.rst

```diff
@@ -1,22 +1,1167 @@
+
+
+.. code-block:: yaml 
+
+  flownet:
+    constraining:
+      kriging:
+        enabled: false
+        n: 20
+        n_lags: 6
+        anisotropy_scaling_z: 10
+        variogram_model: spherical
+        permeability_variogram_parameters:
+          sill: 0.75
+          range: 1000
+          nugget: 0
+        porosity_variogram_parameters:
+          sill: 0.05
+          range: 1000
+          nugget: 0
+
+
+data_source
+-----------
+
+FlowNet pulls data from various sources to create a model of flow tubes. Currently the only data source implemented is an existing model, 
+but this can be extended to extracting data from various continously updated databases.
+
+simulation
+~~~~~~~~~~
+
+FlowNet will extract the data used to construct and condition the model from an existing input simulation model. 
+
+* **input_case**: Path to the input simulation case. 
+* **vectors**: Which vectors to extract from the input simulation case and use in the conditioning. The vectors available are 
+  *WTHP*, *WBHP*, *WOPT*, *WGPT*, *WWPT*, *WWIT*, *WGIT*, *WOPR*, *WGPR*, *WWPR*, *WWIR*, *WGIR*. Each of the vectors added to the config 
+  yaml needs a **rel_error** (relative error, defining e.g. 0.1 will yield a 10% relative error) and **min_error** (minimum error, the 
+  lowest possible observation error) defined.
+* **well_logs**: Boolean variable to indicate if well logs should be loaded from the input simulation model (ADD MORE) 
+  (no default value, not defining it will act the same way as False would)
+* **layers**: If the input simulation model is layered to the extent that there is no (or very little) communication between layers, 
+  FlowNet has an option to generate separate FlowNet models for each layer. To initiate this, supply a list of lists containing the 
+  start and end layer in the input simulation model for each distinct layer
+
+Example yaml section:
+
+.. code-block:: yaml 
+
+  flownet:
+    data_source:
+      simulation:
+        input_case: /path/to/simulation_model.DATA
+        vectors:
+          WOPR:
+            rel_error: 0.1
+            min_error: 50
+          WGPR:
+            rel_error: 0.1
+            min_error: 50
+        layers:
+          - [1, 5]
+          - [6, 10]
+
+In this example, the input simulation model (which has been simulated with Flow or Eclipse or similar) will be found in 
+*/path/to/simulation_model.DATA*, the vectors to use in the conditioning of the FlowNet model are *WOPR* and *WGPR*, each
+with a relative error of 10% and minimum error of 50 (Sm3). Two FlowNet models will be created, one based on layers 1 to 5 
+in the input simulation model, and one based on layers 6 to 10 in the input simulation model.
+
+resampling
+~~~~~~~~~~
+
+Requested observation frequency, all pandas resampling options are supported, e.g. weekly (W), monthly (M), 
+quarterly (Q), yearly (A). If resampling is not defined, the original data will be kept.
+
+
+concave_hull
+~~~~~~~~~~~~
+
+ADD MORE HERE
```

Comment:
In `_config_parser.py` it is stated that 
"When true, the bounding boxes of the gridcells of the original reservoir model are used to check if the generated additional nodes are positioned within the reservoir volume."
I think this is good enough (but I haven't checked if it's correct or not!)